app/src/test/java/org/greenstand/android/TreeTracker/models/ConvergenceTest.kt

```diff
@@ -52,17 +51,6 @@ class ConvergenceTest {
         convergence = Convergence(locations.subList(0, 5))
     }
 
-    @Test
-    fun noConvergenceUntilThresholdDataSize() {

```

Comment:
This test is no longer is relevant for Convergence object given the comment I left for the change in Convergence object above. The responsibility of passing the data with the required size is with the caller.